tools/WinObjC.Packaging/project.json

```diff
@@ -1,8 +1,7 @@
 ﻿{
   "dependencies": {
     "GitInfo": "1.1.32",
-    "NuGet.Build.Packaging": "0.1.186",
-    "WinObjC.Language": "0.2.160915-*"
```

Comment:
Yeah, I meant to make this dependency outside of the project.json. Take it out for now and I'll file a bug and do it another way.

---
In reply to: [102289368](https://github.com/Microsoft/WinObjC/pull/1971#discussion_r102289368) [](ancestors = 102289368)